website/docs/r/policy_gateway_flood_protection_profile.html.markdown

```diff
@@ -64,6 +64,8 @@ The following arguments are supported:
 * `description` - (Optional) Description of the resource.
 * `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
 * `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the policy resource.
+* `context` - (Optional) The context which the object belongs to
+  * `project_id` - (Required) The ID of the project which the object belongs to
```

Comment:
extra space ?

Reply:
before star (*)

Reply:
This is intended indentation, since `project_id` is under `context`. f.e.:
```
context {
  project_id = "proj1"
}
```